lib/oga/xml/document.rb

```diff
@@ -57,6 +57,12 @@ def root_node
         self
       end
 
+      # Returns the top level element in the document.
+
```

Comment:
@scotchi This is intentional. HTML5 documents don't require the usual top-level elements, meaning you can have multiple root elements.

Come to think of it, because of that I'm not entirely sure if having this method makes a lot of sense since it wouldn't work very well for HTML5 documents.